src/amo/routes.js

```diff
@@ -30,6 +32,8 @@ import ServerError from './components/ErrorPage/ServerError';
 export default (
   <Route path="/:lang/:application" component={App}>
     <IndexRoute component={Home} />
+    <Route path="about" component={About} />
+    <Route path="review-guide" component={ReviewGuide} />
```

Comment:
This router config will result in the about page being under `/en-US/firefox/about` which will also mean there's a `/en-US/android/about` too which would result in duplicate content.

It needs to use purely a locale prefix so that you end up with: `/en-US/about` without the `clientApp` (firefox/android) part. 

This will probably require changes to the routes to have more than one `Route` component.

We also still a redirect for `review_guide` to `review-guide`.

Reply:
Maybe this could be the way to configure the new routes:

```
diff --git a/src/amo/routes.js b/src/amo/routes.js
index af60673..6b4e912 100644
--- a/src/amo/routes.js
+++ b/src/amo/routes.js
@@ -32,8 +32,8 @@ import ServerError from './components/ErrorPage/ServerError';
 export default (
   <Route path="/:lang/:application" component={App}>
     <IndexRoute component={Home} />
-    <Route path="about" component={About} />
-    <Route path="review-guide" component={ReviewGuide} />
+    <Route path="/:lang/about/" component={About} />
+    <Route path="/:lang/review-guide/" component={ReviewGuide} />
     <Route path="addon/:slug/" component={Addon} />
     <Route path="addon/:addonSlug/reviews/" component={AddonReviewList} />
     <Route path=":visibleAddonType/categories/" component={CategoriesPage} />
```

Reply:
hi @willdurand, 
`<Route path="/:lang/about/" component={About} />`
`<Route path="/:lang/review-guide/" component={ReviewGuide} />`
doesn't helps
On `http://localhost:3000/en-GB/about/`, I get this
`This page does not exist in addons-frontend. Returning 404; this error should trigger upstream (usually addons-server) to return a valid response.`

Reply:
See https://github.com/mozilla/addons-frontend/issues/3162 which is a blocker.